### PR TITLE
Issue 1940

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

File: src/api/spec/controllers/public_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+require 'webmock/rspec'
```
Comment (hennevogel) [inline src/api/spec/controllers/public_controller_spec.rb]: Why?

Comment (bgeuken) [inline src/api/spec/controllers/public_controller_spec.rb]: That's a leftover from trying a different approach. I'll remove it. Thanks!

File: src/api/spec/factories/packages.rb [hunk 2]
```diff
@@ -1,10 +1,12 @@
 FactoryGirl.define do
   factory :package do
+    project
     sequence(:name) { |n| "package_#{n}" }
     factory :package_with_file do
       after(:create) do |package|
         # NOTE: Enable global write through when writing new VCR cassetes.
         # ensure the backend knows the project
+CONFIG['global_write_through'] = true
```
Comment (mdeniz) [inline src/api/spec/factories/packages.rb]: Is this a leftover? If you needed it at developing time in the controller spec... then let it there commented like in other specs that we have done

Comment (bgeuken) [inline src/api/spec/factories/packages.rb]: It's a leftover. I am currently trying to figure out why this http request is failing even though I created VCR cassettes for it.

I should have closed the PR for now. Sorry for the noise.

**Pull request closed without merging**